Formula/s/surelog.rb

```diff
@@ -0,0 +1,59 @@
+class Surelog < Formula
+  include Language::Python::Virtualenv
+
+  desc "SystemVerilog Pre-processor, parser, elaborator, UHDM compiler"
+  homepage "https://github.com/chipsalliance/Surelog"
+  url "https://github.com/chipsalliance/Surelog/archive/refs/tags/v1.74.tar.gz"
+  sha256 "0fb8d6e55ed3189ecb9aa3e33616d5b4b92f91fa25ff2e4415e1ee661d75fa88"
+  license "Apache-2.0"
+  head "https://github.com/chipsalliance/Surelog.git", branch: "master"
+
+  depends_on "antlr" => :build
+  depends_on "cmake" => :build
+  depends_on "nlohmann-json" => :build
+  depends_on "openjdk" => :build
+  depends_on "python@3.11" => :build
+  depends_on "six" => :build
+  depends_on "googletest" => :test
+  depends_on "pkg-config" => :test
+  depends_on "antlr4-cpp-runtime"
+  depends_on "capnp"
+  depends_on "uhdm"
+
+  resource "orderedmultidict" do
+    url "https://files.pythonhosted.org/packages/53/4e/3823a27d764bb8388711f4cb6f24e58453e92d6928f4163fdb01e3a3789f/orderedmultidict-1.0.1.tar.gz"
+    sha256 "04070bbb5e87291cc9bfa51df413677faf2141c73c61d2a5f7b26bea3cd882ad"
+  end
+
+  def install
+    venv = virtualenv_create(buildpath/"venv", "python3.11")
+    resources.each do |r|
+      venv.pip_install r
+    end
+
+    # Get antlr path
+    #
+    # Build shared library
+    system "cmake", "-S", ".", "-B", "build_shared",
+      "-DBUILD_SHARED_LIBS=ON",
```

Comment:
Isn't this in the std_cmake_args?

Reply:
No. I am following the (well reviewed and scrutinized) pattern of https://github.com/Homebrew/homebrew-core/pull/139476.